src/base/bittorrent/session.h

```diff
@@ -330,6 +334,16 @@ namespace BitTorrent
         void setDiskCacheTTL(uint ttl);
         bool useOSCache() const;
         void setUseOSCache(bool use);
+        bool guidedReadCache() const;
```

Comment:
>**Naming Boolean Getters, Setters, and Properties**
Finding good names for the getter and setter of a bool property is always a special pain. Should the getter be called checked() or isChecked()? scrollBarsEnabled() or areScrollBarEnabled()?
>
In Qt 4, we used the following guidelines for naming the getter function:
>
Adjectives are prefixed with is-. Examples:
isChecked()
isDown()
isEmpty()
isMovingEnabled()
However, adjectives applying to a plural noun have no prefix:
scrollBarsEnabled(), not areScrollBarsEnabled()
Verbs have no prefix and don't use the third person (-s):
acceptDrops(), not acceptsDrops()
allColumnsShowFocus()
Nouns generally have no prefix:
autoCompletion(), not isAutoCompletion()
boundaryChecking()
Sometimes, having no prefix is misleading, in which case we prefix with is-:
isOpenGLAvailable(), not openGL()
isDialog(), not dialog()
(From a function called dialog(), we would normally expect that it returns a QDialog.)
The name of the setter is derived from that of the getter by removing any is prefix and putting a set at the front of the name; for example, setDown() and setScrollBarsEnabled(). The name of the property is the same as the getter, but without the is prefix.

So `useOSCache` kind of makes sense. Even `guidedReadCache` can be used as bool property name.